docs/src/bpmn-workflows/receive-tasks/receive-tasks.md

```diff
@@ -14,6 +14,8 @@ A message can published using one of the Zeebe clients. When the message is corr
 
 A message can be referenced by one or more receive tasks. It **must** define the name of the message (e.g. `Money collected`) and the `correlationKey` expression (e.g. `= orderId`).
 
+Usually, the name of the message is defined as a static value (e.g. `order canceled`) but it can also be defined as [expression](/reference/expressions.html) (e.g. `= "order " + awaitingAction`). The expression is evaluated on activating the receive task and must result in a `string`.
```

Comment:
```suggestion
Usually, the name of the message is defined as a static value (e.g. `order canceled`), but it can also be defined as [expression](/reference/expressions.html) (e.g. `= "order " + awaitingAction`). The expression is evaluated on activating the receive task and must result in a `string`.
```